api/envoy/extensions/rbac/custom_library_config/v3/custom_library.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.extensions.rbac.custom_library_config.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.rbac.custom_library_config.v3";
+option java_outer_classname = "CustomLibraryProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: RBAC CEL Custom Library]
+
+// [#extension: envoy.rbac.custom_library_config.custom_library]
```

Comment:
I think the custom CEL library isn't restricted to RBAC usage, so we should name it envoy.expr.custom_library_config.custom_library or similar (whatever is appropriate based on convention).   e.g. the CEL matcher for the filter in https://github.com/envoyproxy/envoy/issues/18064 will support extensible CEL as well.

Reply:
+1